package.json

```diff
@@ -41,16 +41,17 @@
     "test": "mocha -R spec"
   },
   "dependencies": {
-    "strong-trace": "^1.2.0",
     "debug": "^2.0.0",
     "dotenv": "~0.2.8",
     "posix-getopt": "^1.1.0",
     "strong-agent": "^1.1.1",
     "strong-cluster-control": "^2.0.0",
     "strong-control-channel": "^1.1.0",
+    "strong-license": "~1.0.0",
```

Comment:
That's the wrong package name (you use `strongloop-license`), but it doesn't matter, you'll move the depencency to strong-trace.

```
sn/strong-supervisor (master u=) % git co trace-license-check
Branch trace-license-check set up to track remote branch trace-license-check from origin.
Switched to a new branch 'trace-license-check'
sn/strong-supervisor (trace-license-check u=) % npm install
strong-license@1.0.2 node_modules/strong-license
└── jwt-simple@0.2.0
sn/strong-supervisor (trace-license-check u=) % ./bin/sl-run.js test/module-app/
INFO strong-agent API key not found, StrongOps dashboard reporting disabled.
Generate configuration with:
    npm install -g strongloop
    slc strongops
See http://docs.strongloop.com/strong-agent for more information.
INFO strong-agent v1.3.2 profiling app 'module-app' pid '25302'
INFO strong-agent[25302] started profiling agent
module.js:318
    throw err;
          ^
Error: Cannot find module 'strongloop-license'
    at Function.Module._resolveFilename (module.js:316:15)
    at Function.Module._load (module.js:258:25)
    at Module.require (module.js:345:17)
    at Module.module.__proto__.require (/home/sam/w/sn/strong-supervisor/node_modules/strong-agent/lib/agent.js:237:42)
    at require (module.js:364:17)
    at Object.<anonymous> (/home/sam/w/sn/strong-supervisor/bin/sl-run.js:57:5)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:428:10)
    at Module.load (module.js:335:32)
    at Function.Module._load (module.js:290:12)
```